modules/installation-creating-aws-control-plane.adoc

```diff
@@ -150,58 +150,7 @@ directory. This value is the long string with the format
 <20> Specify the `MasterInstanceProfile` parameter value from the output of
 the CloudFormation template for the security group and roles.
 <21> The type of AWS instance to use for the control plane machines.
-<22> Allowed values:
-* `m4.xlarge`
-* `m4.2xlarge`
-* `m4.4xlarge`
-* `m4.10xlarge`
-* `m4.16xlarge`
-* `m5.xlarge`
-* `m5.2xlarge`
-* `m5.4xlarge`
-* `m5.8xlarge`
-* `m5.12xlarge`
-* `m5.16xlarge`
-* `m5a.xlarge`
-* `m5a.2xlarge`
-* `m5a.4xlarge`
-* `m5a.8xlarge`
-* `m5a.12xlarge`
-* `m5a.16xlarge`
-* `c4.2xlarge`
-* `c4.4xlarge`
-* `c4.8xlarge`
-* `c5.2xlarge`
-* `c5.4xlarge`
-* `c5.9xlarge`
-* `c5.12xlarge`
-* `c5.18xlarge`
-* `c5.24xlarge`
-* `c5a.2xlarge`
-* `c5a.4xlarge`
-* `c5a.8xlarge`
-* `c5a.12xlarge`
-* `c5a.16xlarge`
-* `c5a.24xlarge`
-* `r4.xlarge`
-* `r4.2xlarge`
-* `r4.4xlarge`
-* `r4.8xlarge`
-* `r4.16xlarge`
-* `r5.xlarge`
-* `r5.2xlarge`
-* `r5.4xlarge`
-* `r5.8xlarge`
-* `r5.12xlarge`
-* `r5.16xlarge`
-* `r5.24xlarge`
-* `r5a.xlarge`
-* `r5a.2xlarge`
-* `r5a.4xlarge`
-* `r5a.8xlarge`
-* `r5a.12xlarge`
-* `r5a.16xlarge`
-* `r5a.24xlarge`
+<22> Suggested values: `<InstanceType>.xlarge`, where `InstanceType` could be: `m5`,`m6i`, etc
```

Comment:
I think users may get confused, why only `*.xlarge` machines are suggested, what about `*.4xlarge`.
I suggest using a general description, e.g.
_the instance type could be: m5.xlarge, m6i.xlarge, etc._

Reply:
@

> I think users may get confused, why only `*.xlarge` machines are suggested, what about `*.4xlarge`. I suggest using a general description, e.g. _the instance type could be: m5.xlarge, m6i.xlarge, etc._

@yunjiang29 what about?:

```diff
-<22> Suggested values: `<InstanceType>.xlarge`, where `InstanceType` could be: `m5`,`m6i`, etc
+<22> The minimal size is `xlarge`. The instance type could be: `m5.2xlarge`, `m6i.4xlarge`, etc.
```

Reply:
No, it is independent of whether it is `xlarge` or not, that's different concepts.

Reply:
@yunjiang29 yes, `xlarge` applies only for minimal CPU requirements for control planes, I forgot a few EC2 types, it did not match the minimal RAM requirements, like `c*.xlarge` which is ~8GiB. So I agree we could leave a more generic message here. Feel free to suggest. Let's talk on Slack, just left a message for you there. :)

Reply:
@yunjiang29 updated to generic value as our conversation on Slack.